### PR TITLE
[WIP] Document migration from integrated to external cloud provider

### PR DESCRIPTION
**Pull request by onitake**
**Note: This is work in progress. Do not merge yet.**

Addresses https://github.com/kubernetes/cloud-provider/issues/42
CC @andrewsykim

### CODE REVIEW

**Comment by sftim** [comment]
/uncc
for now. Happy to take another look once it's ready for review.

File: content/en/docs/tasks/administer-cluster/migrating-to-cloud-controller-manager.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+reviewers:
+- andrewsykim
+title: Migrating to Cloud Controller Manager
+content_template: templates/task
+---
```
[the remark this answers is not in the document]
Comment (onitake) [reply content/en/docs/tasks/administer-cluster/migrating-to-cloud-controller-manager.md]: Will do - actually, and I think it should be 1.16.
On the other hand, I will probably remove `{{< include "task-tutorial-prereqs.md" >}}`, as that should be obvious when doing a KCM->CCM migration, or rewrite.

**Comment by andrewsykim** [comment]
Sorry for the delay, going to get to this soon

**Comment by onitake** [comment]
@andrewsykim I haven't been able to continue working on it since my last commit.
It's still highly incomplete and lacks precise steps for specific environments.

I also realized that sectioning per step first, then per environment makes it very unreadable.
I'm considering flipping this around, with high-level sections per environment type, then different subsections per step.
What do you think?

**Comment by VineethReddy02** [comment]
👋 Please rebase this PR on `dev-1.18` in order to avoid merge conflicts.

Feel free to `/hold cancel` after you've rebased.

/hold

**Comment by VineethReddy02** [comment]
Hi @onitake 
As we are close to the 1.18 release. Can we have these changes in place asap? If this is targeted for 1.18 release.

**Comment by onitake** [comment]
@VineethReddy02 The changes aren't ready yet, this is just a preliminary document. I'm even considering a complete rewrite.

If you are close to 1.18 lockdown, I think the milestone should be replaced.

**Comment by VineethReddy02** [comment]
Hi, @onitake Thanks for the quick update, in that case, this needs to be targeted to the future branch.

**Comment by VineethReddy02** [comment]
Hi @onitake As 1.18 release is tomorrow i.e 24th March. We are not accepting any docs for the 1.18 release.
Can you please close this PR?

**Comment by onitake** [comment]
All right, I will retarget dev-1.19. Hopefully I can get it finished until then.

**Pull request closed without merging**